Sources/WalletConnectSign/Storage/SequenceStore.swift

```diff
@@ -54,6 +54,12 @@ final class SequenceStore<T> where T: ExpirableSequence {
         storage.removeObject(forKey: getKey(for: topic))
     }
     
+    func deleteAll() {
+        for key in storage.dictionaryRepresentation().keys {
+            storage.removeObject(forKey: key)
+        }
+    }
```

Comment:
Added logic what delete values for current `identifier` only. 
Also improved filtration by `identifier`. Now it works for "test" and "test2" identifiers too, because they starts with "test"